cypress/integration/publishingFlows/articleBrowserPersistence.spec.js

```diff
@@ -0,0 +1,195 @@
+describe('Article Editor', () => {
+  describe('v1 Editor', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/new');
+        });
+      });
+    });
+
+    describe(`revert changes`, () => {
+      it('should revert to the initial v1 editor template if it is a new article', () => {
+        cy.findByRole('form', { name: /^Edit an article$/i }).as('articleForm');
+
+        // Fill out the title, tags, and content for an article.
+        cy.get('@articleForm')
+          .findByLabelText('Post Content')
+          .as('postContent')
+          // Clearing out the whole content area as this seemed simpler than finding where to add the fields in the v1 editor
+          .clear()
+          // The v1 editor has all the post info in one textarea
+          .type(
+            `---\ntitle: \npublished: true\ndescription: some description\ntags: tag1, tag2,tag3\n//cover_image: https://direct_url_to_image.jpg\n---\n\nThis is some text that should be reverted`,
+          );
+
+        cy.get('@postContent').should(
+          'have.value',
+          `---\ntitle: \npublished: true\ndescription: some description\ntags: tag1, tag2,tag3\n//cover_image: https://direct_url_to_image.jpg\n---\n\nThis is some text that should be reverted`,
+        );
+
+        cy.findByRole('button', { name: /^Revert new changes$/i }).click();
+
+        // The article editor should reset to it's initial values
+        cy.get('@postContent').should(
+          'have.value',
+          `---\ntitle: \npublished: false\ndescription: \ntags: \n//cover_image: https://direct_url_to_image.jpg\n---\n\n`,
```

Comment:
suggestion (not blocking) - I'm wondering if this tests would be more easily readable if we extracted this 'initial state' text into a `const` e.g. so we have

```js
.type(`${initialText}\n\nThis is some text that should be reverted`);
...
cy.get('@postContent').should('have.value', initialText);
```